examples/vitess/vtctld-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: vtctld
+  labels:
+    name: vtctld
+spec:
+  ports:
+    - port: 15000
+      targetPort: 15000
+      nodePort: 30000
```

Comment:
I tried to make this work, but the problem is opening the port in the firewall. Since (afaik) there's no automated way to do that across platforms (e.g. GCE vs AWS), I'm telling the user to do that manually and pointing them to some docs.

If I don't know the port in advance, `vitess-up.sh` can't do everything it needs without user intervention, which defeats the purpose. I understand you're interested in splitting it up in a later iteration, but for now I don't think making the port automatic is worth compromising the one-click goal of the current incarnation.

Let me know if you feel strongly about it. In that case, I would probably have to do some work to split the tutorial up to be closer to your ideal granularity before I can make the port automatic.